submissions/vovan-zt/popup/index.html

```diff
@@ -0,0 +1,71 @@
+                            <li class="modal__list"><a href="#"> <img src="icons/contacts.png" alt="img"><span>item7</span></a></li>
+                            <li class="modal__list"><a href="#"> <img src="icons/documents.png" alt="img"><span>item8</span></a></li>
+                            <li class="modal__list"><a href="#"> <img src="icons/files.png" alt="img"><span>item9</span></a></li>
+                            <li class="modal__list"><a href="#"> <img src="icons/mail.png" alt="img"><span>item10</span></a></li>
+                            <li class="modal__list"><a href="#"> <img src="icons/maps.png" alt="img"><span>item11</span></a></li>
+                            <li class="modal__list"><a href="#"> <img src="icons/mars.png" alt="img"><span>item12</span></a></li>
+                            
+                            <input  class="popup__check_input" type="checkbox" name="" id="modal2">
+                            <label  for="modal2" class="popup__link2"></label> 
+        
+                            <li class="modal__list modal__list2" ><a href="#"><img src="icons/collection.png" alt="img"> <span>item13</span></a></li>
+                            <li class="modal__list modal__list2" ><a href="#"><img src="icons/contacts.png" alt="img"> <span>item14</span></a></li>
+                            <li class="modal__list modal__list2" ><a href="#"><img src="icons/documents.png" alt="img"> <span>item15</span></a></li>
+                            <li class="modal__list modal__list2" ><a href="#"><img src="icons/files.png" alt="img"><span>item16</span></a></li>
+                            <li class="modal__list modal__list2" ><a href="#"><img src="icons/mail.png" alt="img"><span>item17</span></a></li>
+                            <li class="modal__list modal__list2" ><a href="#"><img src="icons/maps.png" alt="img"><span>item18</span></a></li>
+                            <li class="modal__list modal__list2" ><a href="#"><img src="icons/mars.png" alt="img"><span>item19</span></a></li>
+                            <li class="modal__list modal__list2" ><a href="#"><img src="icons/maps.png" alt="img"><span>item20</span></a></li>
+                            <li class="modal__list modal__list2" ><a href="#"><img src="icons/mars.png" alt="img"><span>item21</span></a></li>
+                        </ul>
+                    </div> 
+                </li>
+                <li  class="popup__list"> 
+                    <input class="popup__check_input" type="checkbox" >
+                    <label><img class="popup__link" src="icons/bell.png" alt="img"></label> 
+                </li>
+                <li  class="popup__list">
+                    <input class="popup__check_input" type="checkbox" >
+                    <label><img class="popup__link" src="icons/bell.png" alt="img"></label> 
+                </li>
+            </ul>
+        </nav>
+    </div>
+
+</body>
+</html>
```

Comment:
Add an empty lines at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)